onnxruntime/python/tools/quantization/test/test_calibrate.py

```diff
@@ -7,130 +7,277 @@
+        added_node_names = ['C_ReduceMin', 'C_ReduceMax', 'D_ReduceMin', 'D_ReduceMax']
+        added_outputs = ['C_ReduceMin', 'C_ReduceMax', 'D_ReduceMin', 'D_ReduceMax']
+        # Original 3 nodes + added ReduceMin/Max nodes * 4 (exlude graph input/output)
         self.assertEqual(len(augmented_model_node_names), 7)
-        # original single graph output with 4 added ones
+        # Original 1 graph output + added outputs * 4
         self.assertEqual(len(augmented_model_outputs), 5)
         for name in added_node_names:
             self.assertTrue(name in augmented_model_node_names)
         for output in added_outputs:
             self.assertTrue(output in augmented_model_outputs)
 
-        # Checking order of augmented graph outputs
+        print('Finished TEST_CONFIG_1')
+
+
+        '''TEST_CONFIG_2'''
+
+        #   Conv
+        #    |   
+        #   Conv
+
         G = helper.make_tensor_value_info('G', TensorProto.FLOAT, [1, 1, 5, 5])
-        H = helper.make_tensor_value_info('H', TensorProto.FLOAT, [1, 1, 5, 1])
-        I = helper.make_tensor_value_info('I', TensorProto.FLOAT, [1, 1, 5, 1])
-        J = helper.make_tensor_value_info('J', TensorProto.FLOAT, [1, 1, 5, 1])
-        K = helper.make_tensor_value_info('K', TensorProto.FLOAT, [1, 1, 5, 1])
-        matmul_node_1 = onnx.helper.make_node('MatMul', ['G', 'H'], ['I'], name='MatMul_1')
-        matmul_node_2 = onnx.helper.make_node('MatMul', ['I', 'J'], ['K'], name='MatMul_2')
-        graph = helper.make_graph([matmul_node_1, matmul_node_2], 'test_graph_order', [G, H, J], [K])
+        H = helper.make_tensor_value_info('H', TensorProto.FLOAT, [1, 1, 3, 3])
+        I = helper.make_tensor_value_info('I', TensorProto.FLOAT, [1, 1, 5, 5])
+        J = helper.make_tensor_value_info('J', TensorProto.FLOAT, [1, 1, 3, 3])
+        K = helper.make_tensor_value_info('K', TensorProto.FLOAT, [1, 1, 5, 5])
+        conv_node_1 = onnx.helper.make_node('Conv', ['G', 'H'], ['I'], name='Conv', kernel_shape=[3, 3], pads=[1, 1, 1, 1])
+        conv_node_2 = onnx.helper.make_node('Conv', ['I', 'J'], ['K'], name='Conv', kernel_shape=[3, 3], pads=[1, 1, 1, 1])
+        graph = helper.make_graph([conv_node_1, conv_node_2], 'test_graph_2', [G, H, J], [K])
+        model = helper.make_model(graph)
+        test_model_path = './test_model_2.onnx'
+        onnx.save(model, test_model_path)
+
+        # Augmenting graph
+        data_reader = TestDataReader()
+        augmented_model_path = './augmented_test_model_2.onnx'
+        calibrater = ONNXCalibrater(test_model_path, data_reader, ['Conv', 'MatMul'], [], [], augmented_model_path)
+        augmented_model = calibrater.augment_graph()
+        onnx.save(augmented_model, augmented_model_path)
+
+        augmented_model_node_names = [node.name for node in augmented_model.graph.node]
+        augmented_model_outputs = [output.name for output in augmented_model.graph.output]
+        added_node_names = ['I_ReduceMin','I_ReduceMax']
```

Comment:
>added_node_names [](start = 8, length = 16)

The last Conv also needs to be calibrated or our current quantization will throw an error. The fix is here: [https://github.com/microsoft/onnxruntime/blob/c0d7c8bc44185e02485ec787b7ba73e1435508fc/onnxruntime/python/tools/quantization/calibrate.py#L59](https://github.com/microsoft/onnxruntime/blob/c0d7c8bc44185e02485ec787b7ba73e1435508fc/onnxruntime/python/tools/quantization/calibrate.py#L59). Thus, the K value will also be calibrated. 
Looks like your code is old and please update to the latest master.

Reply:
Got it. Yes, the pr was sent out several days ago, the code is old.